stack/keycloak/migration/add-authorities-wahlvorbereitung-unterbrechungsuhrzeit.yml

```diff
@@ -0,0 +1,39 @@
+id: add authorities wahlvorbereitung unterbrechungsuhrzeit
```

Comment:
muss noch in `keycloak-changelog.yml` referenziert werden

Reply:
resolved with a131f9dc5a669bb316108a496726bc68a50216d4